scripts/test_updates_pg13.sh

```diff
@@ -10,3 +10,6 @@ source ${SCRIPT_DIR}/test_functions.inc
 run_tests -v7 \
           2.1.0-pg13 2.1.1-pg13 2.2.0-pg13 2.2.1-pg13 2.3.0-pg13 2.3.1-pg13 \
           2.4.0-pg13 2.4.1-pg13 2.4.2-pg13 2.5.0-pg13 2.5.1-pg13
```

Comment:
```suggestion
          2.4.0-pg13 2.4.1-pg13 2.4.2-pg13
```
v8 should be a superset of v7 and if it works starting at 2.5.0 then  2.5+ should be removed from v7 here.